docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/index.md

```diff
@@ -1,316 +1,68 @@
 ---
 title: "BigQuery Loader"
-sidebar_position: 2
+sidebar_label: "BigQuery Loader"
+sidebar_position: 2 
 ---
 
 ```mdx-code-block
-import {versions} from '@site/src/componentVersions';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 import CodeBlock from '@theme/CodeBlock';
-import Diagram from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/_diagram.md';
+import LoaderDiagram from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/_diagram.md';
+import DeployOverview from '@site/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/_deploy-overview.md';
 ```
 
-Under the umbrella of Snowplow BigQuery Loader, we have a family of applications that can be used to load enriched Snowplow data into BigQuery.
-
-<Diagram/>
+## Overview
+
+The BigQuery Loader is an application that loads Snowplow events to BigQuery using the [BigQuery Storage API](https://cloud.google.com/bigquery/docs/write-api).
```

Comment:
Could we avoid repetition here?

<img width="730" alt="image" src="https://github.com/snowplow/documentation/assets/2670454/ed22ee0c-5f33-4364-9c14-b894d7a1feed">